Packager/PackagerImplementation.cpp

```diff
@@ -26,6 +26,10 @@
 #endif
 #include <opkg_download.h>
 
+#include <cjson/cJSON.h>
```

Comment:
For clear visibility on external dependencies, Thunder development rule, put external header files (encompassed by the '<>' characters,) in the Module.h file. This way dependencies of a plugin can be easilly tracked.